draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -217,13 +203,42 @@ The TTL field's value is not defined by this memo.  Catalog zones are
 for authoritative nameserver management only and are not intended for general
 querying via recursive resolvers.
 
-# Properties {#properties}
+
+## Global Properties
+
+Apart from catalog zone metadata stored at the apex (NS, SOA and the like), catalog zone information is stored in the form of "properties".
+
+This specification defines a number of so-called properties that implementations MUST support,
+as well as a mechanism to allow implementors to store additional information in the catalog zone (= extension properties, (#customproperties)).
+The meaning of such extension properties is determined by the implementation in question.
+
+Some properties are defined at the global level; others are scoped to apply only to a specific member zone.
+This section deals with global properties; member-specific properties are described in (#properties).
+
+More properties may be defined in future documents.
+
+### Schema Version (`version` property)
+
+The catalog zone schema version is specified by an integer value embedded in a TXT RR named `version.$CATZ`.
+All catalog zones MUST have a TXT RRset named `version.$CATZ` with at least one RR.
+Catalog consumers MUST NOT apply catalog zone processing to zones without the expected value in one of the RRs in the `version.$CATZ` TXT RRset, but they may be transferred as ordinary zones.
+For this memo, the value of one of the RRs in the `version.CATZ` TXT RRset MUST be set to "2", i.e.
+
+``` dns-zone
+version.$CATZ 0 IN TXT "2"
+```
```

Comment:
A global ext property is an incompatible change. I think we should consider bumping the version since processing vs ignoring the global ext property might lead to unexpected results.

Reply:
I don't this is necessary.

In the Description section it states: "Catalog consumers SHOULD ignore any RR in the catalog zone which is meaningless or useless to the implementation.", so current version 2 implementation should already ignore global `.ext` properties.

Furthermore, in Section Custom Properties (*.ext properties): "The meaning of the custom properties described in this section is determined by the implementation alone, without expectation of interoperability."

Also, we haven't had any global property yet (except for the `version` property).

@mind04 @Habbie @libor-peltan-cznic @peterthomassen you agree?

Reply:
I think we should stick with version "2".

Reply:
"2" works for me.